Dockerfile

```diff
@@ -26,8 +30,13 @@ RUN apt-get update && apt-get install -y unzip --no-install-recommends && \
 RUN chmod +x /usr/bin/dep
 
 WORKDIR /go/src/github.com/darthhater/bored-board-service
+
 COPY . .
 
+RUN mkdir .keys && \
+    openssl genrsa -out .keys/app.rsa 1024 && \
```

Comment:
This will need modified to:
`openssl genrsa -out ${PRIVATE_KEY_PATH} 1024 && \`

Suggest setting your default for the ENV or ARG to the path rather then `default` as it's going to likely error with that value